Hudl.Mjolnir/Command/Command.cs

```diff
@@ -38,6 +38,11 @@ public abstract class Command
         protected static readonly ConfigurableValue<bool> UseCircuitBreakers = new ConfigurableValue<bool>("mjolnir.useCircuitBreakers", true);
 
         /// <summary>
+        /// If this is set to true then all calls wrapped in a Mjonir command will ignore the default timeout.
```

Comment:
I would leave a comment here on *why* someone might want to use this config (basically for testing), and that it's not advised to use it for production.